pkg/server/commands/reverseexpand/reverse_expand_test.go

```diff
@@ -443,3 +447,277 @@ func TestReverseExpandIgnoresInvalidTuples(t *testing.T) {
 		}
 	}
 }
+
+func TestReverseExpandThrottle(t *testing.T) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
+
+	model := testutils.MustTransformDSLToProtoWithID(`model 
+	schema 1.1
+
+	type user
+
+	type document
+	  relations
+		define viewer: [user]`)
+	mockController := gomock.NewController(t)
+	defer mockController.Finish()
+	mockDatastore := mocks.NewMockOpenFGADatastore(mockController)
+
+	ctx := context.Background()
+	ts, err := typesystem.NewAndValidate(ctx, model)
```

Comment:
```suggestion
	typesys, err := typesystem.NewAndValidate(ctx, model)
```
ts is reserved elsewhere in our code for meaning `timestamp`. It's a little confusing to use that shorthand here. `typesys` is more common